Service/src/app.py

```diff
@@ -42,13 +42,14 @@ def setup():
 
 
 @app.route('/')
-@app.route('/api/v1.0')
+@app.route('/api')
```

Comment:
why remove the versioning?

Reply:
The endpoints for every route didn't have versioning. It was either adding versions to each of them or removing it once at the app.py

Reply:
ok - this is resolved